server/src/main/java/io/envoyproxy/controlplane/server/StartupConfigs.java

```diff
@@ -0,0 +1,5 @@
+package io.envoyproxy.controlplane.server;
```

Comment:
another option is put this in `cache` repo. but feel more reasonable to have it on `server` level